jquery.daterangepicker.js

```diff
@@ -433,14 +433,15 @@
 		// expose some api
 		$(this).data('dateRangePicker',
 		{
-			setDateRange : function(d1,d2)
+			setDateRange : function(d1,d2,silent)
 			{
 				if (typeof d1 == 'string' && typeof d2 == 'string')
 				{
 					d1 = moment(d1,opt.format).toDate();
 					d2 = moment(d2,opt.format).toDate();
 				}
-				setDateRange(d1,d2);
+				silent === true ? true : false;

```

Comment:
what does this line mean?   did you mean   ` silent = silent === true ? true : false;`    ?